xcode/Subconscious/Shared/Components/Common/RowButtonStyle.swift

```diff
@@ -7,20 +7,105 @@
 
 import SwiftUI
 
+struct RowLabelStyle: LabelStyle {
```

Comment:
RowButtonStyle now also applies a label style, so that icons are displayed to the right of the row by default.